dev-1/ShopModelTest/DiscountTest.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Core.Discounts;
+using NUnit.Framework;
+
+namespace ShopModelTests
+{
+    [TestFixture]
+    public class DiscountTest
+    {
+        Discount_1 discount_1 = new Discount_1();
+        Discount_5 discount_5 = new Discount_5();
+        const decimal value = 10;
+
+        [Test]
+        public void DiscountOneTest()
+        {
+            var expected = 9.9;
+
+            var percentageValue = discount_1.PercentageValue(value);
```

Comment:
Use direct type initialization in your tests here instead of "var" because we need to know what type is used within this validation step.

Reply:
Apply this change to all your tests